utils/dump_ensembl_valid_species.pl

```diff
@@ -29,7 +30,21 @@ BEGIN
 use LoadPlugins;
 use EnsEMBL::Web::SpeciesDefs;
 
+my $prod = 0;
+
+GetOptions(
+  "production=s" => \$prod,
+);
+
 my $sd = EnsEMBL::Web::SpeciesDefs->new;
-print join "\n", $sd->valid_species, '';
+my @sp_list;
+
+if ($prod eq 'true') {
```

Comment:
From my understanding here, you have to specify `production=true`. Instead of true, maybe better to have `production=1` and then in the code just check for ($prod). cleaner code and also easier command line

Reply:
after discussing with @jyothishnt we decided not to have the extra parameter to dump the species production, in fact the script should be updated to dump the production name by default.

Reply:
To also check if the script is used in our release SOP (jira tickets)

Reply:
Only used in Release SOP to generate taxon tree.

Reply:
Ok thanks for checking, i think we are fine then to go ahead with the suggested changes. Not to have the if statement. And after reviewing Anne PR aswell where she cleanup the species name, its making more sense to remove this one as well.